optaplanner-core/src/main/java/org/optaplanner/core/api/solver/SolverStatus.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.solver;
+
+public enum SolverStatus {
+    SOLVING_SCHEDULED,
+    SOLVING_ACTIVE,
+    NOT_SOLVING;
+}
```

Comment:
Do we want to describe the possible transitions between these?

Reply:
Fixed, thanks for reporting :)